src/register/RegistrationPage.jsx

```diff
@@ -432,6 +432,13 @@ class RegistrationPage extends React.Component {
       );
     }
 
+    if (this.props.registrationResult.success && window.hj) {
+      window.hj('identify', null, {
+        'Signed up': new Date().toISOString(),
+        'Test Account': this.state.username.includes('hotjarTest'), // TODO: Remove this after testing
```

Comment:
Can we always send the username attribute and add that condition on HJ side?

Reply:
I am not sure if we should send any PII, even in test case I am sending whether it is test user or not (true or false)

Reply:
Hmm, makes sense 👍